app/src/androidTest/java/org/simple/clinic/MigrationAndroidTest.kt

```diff
@@ -43,9 +45,14 @@ class MigrationAndroidTest {
   @JvmField
   val helper = MigrationTestHelperWithForeignConstraints()
 
-  @Rule
-  @JvmField
-  val expectedException = ExpectedException.none()
+  private val rxErrorsRule = RxErrorsRule()
+
+  private val expectedException = ExpectedException.none()
+
+  @get:Rule
+  val ruleChain = RuleChain
+      .outerRule(rxErrorsRule)
```

Comment:
There's no RxJava usage in this class, right? Using this rule here is unnecessary.

Reply:
Fixed